src/style.css

```diff
@@ -0,0 +1,109 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  border-radius: 0.5px;
+}
+
+.stars__star:last-of-type {
+  margin-right: 0;
+}
+
+/* STARS--0 */
+
+.stars--0 .stars__star:nth-of-type(1),
+.stars--0 .stars__star:nth-of-type(2),
+.stars--0 .stars__star:nth-of-type(3),
+.stars--0 .stars__star:nth-of-type(4),
+.stars--0 .stars__star:nth-of-type(5) {
+  background-image: url("./images/star.svg");
```

Comment:
Instead of repeating the background styles for each modifier class. You can set the default background-image, background-repeat, and background-position styles once for the .stars__star class and only update the background-image for active stars in the modifier classes. 💡